haskell/haddock.bzl

```diff
@@ -105,24 +124,21 @@ def _haskell_doc_aspect_impl(target, ctx):
             set.to_depset(target[HaskellLibraryInfo].source_files),
             target[HaskellLibraryInfo].extra_source_files,
             depset([
-                hs.tools.bash,
                 hs.tools.ghc_pkg,
                 hs.tools.haddock,
-                hs.tools.mktemp,
-                hs.tools.rmdir,
             ]),
             locale_archive_depset,
         ]),
         outputs = [haddock_file, html_dir],
         mnemonic = "HaskellHaddock",
         progress_message = "HaskellHaddock {}".format(ctx.label),
-        executable = ctx.file._haddock_wrapper,
+        executable = haddock_wrapper,
         arguments = [
             prebuilt_deps,
             args,
             ghc_args,
         ],
-        env = hs.env,
+        use_default_shell_env = True,
```

Comment:
Because the haddock wrapper script use `mktemp` and `rmdir`. Previously, those were "visible binaries". Now, they are taken from the default shell environment (which Bazel will be restricting soon, by flipping `--experimental_strict_action_env` on by default).